src/index.js

```diff
@@ -150,10 +150,22 @@ async function cli () {
     } else {
       output = input ? (`${input}.svg`) : 'out.svg'
     }
-  }
-  if (!/\.(?:svg|png|pdf|md|markdown)$/.test(output)) {
+  } else if (output === '-') {
+    // `--output -` means write to stdout.
+    output = '/dev/stdout'
+    quiet = true
+
+    if (!outputFormat) {
+      outputFormat = 'svg'
+      warn('No output format specified, using svg. ' +
+        'If you want to specify an output format and supress this warning, ' +
+        'please use `-e <format>.` '
+      )
+    }
+  } else if (!/\.(?:svg|png|pdf|md|markdown)$/.test(output)) {
     error('Output file must end with ".md"/".markdown", ".svg", ".png" or ".pdf"')
   }
+
   const outputDir = path.dirname(output)
   if (!fs.existsSync(outputDir)) {
     error(`Output directory "${outputDir}/" doesn't exist`)
```

Comment:
```suggestion
  if (output !== '/dev/stdout') {
    const outputDir = path.dirname(output)
    if (!fs.existsSync(outputDir)) {
      error(`Output directory "${outputDir}/" doesn't exist`)
```

Reply:
That's a good idea.

Non-Linux operating systems probably won't have a `/dev` folder.

If you want to avoid having an extra level of indentation, we could also change the code to the following:

```suggestion
const outputDir = path.dirname(output)
if (output !== '/dev/stdout' && !fs.existsSync(outputDir)) {
  error(`Output directory "${outputDir}/" doesn't exist`)
```

Reply:
Noted.